.github/actions/linux-deps/action.yml

```diff
@@ -6,5 +6,5 @@ runs:
     - run: sudo apt-get update
       shell: bash
 
-    - run: sudo apt-get install libusb-1.0-0-dev libudev-dev
+    - run: sudo apt-get install libusb-1.0-0-dev libudev-dev libsecp256k1
```

Comment:
Not a big deal, although now a days the "user facing" package manager is just `apt`, while `apt-get` can be [described as an internal tool](https://askubuntu.com/questions/829864/is-apt-get-deprecated).